### PR TITLE
Improve language in HMR docs

### PR DESCRIPTION
**Pull request by siddharthkp**
Sometimes, using language which says things are simple might be condescending towards beginners. 

HMR has been amazing for my productivity, but It took me a while to get it right the first time I tried to set it up 😄 

```diff
- Enabling this feature is actually fairly simple.
+ This feature is great for productivity
```

Thanks to @supreetpal for pointing this out.

### CODE REVIEW

**Comment by siddharthkp** [comment]
Umm folks, the CI build has been broken since a6817aea6c4b6e3aaf3b571d03dc6fd37abd5c58

**Comment by TheLarkInn** [comment]
Hmm let's see what we can do to fix this. @skipjack have you seen this?

**Comment by siddharthkp** [comment]
Give me a second, I'll fix it.

Seems like mixed spaces and tabs.

**Comment by supreetpal** [comment]
It would also be nice if we could fix line 116 `Hot Module Replacement with CSS is actually fairly straightforward`.

**Comment by siddharthkp** [comment]
Build issues are being fixed in https://github.com/webpack/webpack.js.org/pull/1413

**Comment by skipjack** [comment]
I'm good with changing this. When I wrote it, I meant was trying to imply that it's actually a lot more simple than most people think but I can see how it could be misconstrued. I might rephrase the changes a little though or simply drop that line -- I'll review shortly.

File: content/guides/hot-module-replacement.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ W> __HMR__ is not intended for use in production, meaning it should only be used
 
 ## Enabling HMR
 
-Enabling this feature is actually fairly simple. Let's take a look at how to set it up with [webpack-dev-server](https://github.com/webpack/webpack-dev-server)...
+This feature is great for productivity. Let's take a look at how to set it up with [webpack-dev-server](https://github.com/webpack/webpack-dev-server)...
```
Comment (skipjack) [inline content/guides/hot-module-replacement.md]: 👍

File: content/guides/hot-module-replacement.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ This is just one example, but there are many others that can easily trip people
 
 ## HMR with Stylesheets
 
-Hot Module Replacement with CSS is actually fairly straightforward with the help of the `style-loader`. This loader uses `module.hot.accept` behind the scenes to patch `<style>` tags when CSS dependencies are updated. So, with the following webpack configuration...
+We take the help of the `style-loader` to achieve Hot Module Replacement with CSS. This loader uses `module.hot.accept` behind the scenes to patch `<style>` tags when CSS dependencies are updated. So, with the following webpack configuration...
```
Comment (skipjack) [inline content/guides/hot-module-replacement.md]: I would rephrase the first sentence as:

> We can use the `style-loader` to achieve Hot Module Replacement with CSS.

**Comment by siddharthkp** [comment]
@skipjack Sorry for taking so long. Updated!
